app/lib/bulk_download_service.rb

```diff
@@ -66,7 +77,7 @@ def self.generate_curl_configuration(study_files:,
     MetricsService.log('file-download:curl-config', {
       numFiles: study_files.count,
       numStudies: studies.count,
-      studyAccesions: studies.map(&:accession),
+      studyAccessions: studies.map(&:accession),
```

Comment:
Good catch!

Is it trivial to get counts for each study file _type_ -- at least for the big three?  E.g., `numMetadataFiles`, `numClusterFiles`, `numExpressionFiles`.  This is something Leyla was analyzing in the context of single-file downloads, and if it's < 10 minutes to instrument that here it might be worthwhile.  (If not, no worries, I can ticket it for later.)

Reply:
Yeah that should be very easy - I can make that change.